corehq/motech/dhis2/events_helpers.py

```diff
@@ -7,6 +11,10 @@
 
 def send_dhis2_event(request, form_config, payload):
     event = get_event(request.domain_name, form_config, payload)
+    try:
+        Schema(get_event_schema()).validate(event)
+    except SchemaError as err:
+        raise ConfigurationError from err
```

Comment:
What would you think of moving all the Schema business into a `validate_event_schema` function which raises `ConfigurationError`?

Reply:
:+1: Good call.